### PR TITLE
Fix integration test for aws and azure k8s

### PR DESCRIPTION
**Pull request by miseyu**
# Summary
- Fixed due to Scalar helm version update
- The custom config file of helm assumes that monitoring is enabled. (https://github.com/scalar-labs/scalar-kubernetes/blob/master/conf/scalardl-custom-values.yaml#L4-L12)

### CODE REVIEW

File: test/src/integration/integration_test.go [hunk 1]
```diff
@@ -167,6 +168,10 @@ func runAnsiblePlaybooks(t *testing.T) {
 	// Install tools
 	runAnsiblePlaybook(t, k8sModuleDir, "../inventories", []string{"./playbooks/playbook-install-tools.yml", "-e", "base_local_directory=../../../../"})
 
+	// Workaround
+	// Because the configuration file assumes that monitoring is enabled
+	runKubectl(t, "create", "ns", "monitoring")
```
Comment (miseyu) [inline test/src/integration/integration_test.go:173]: In this case, we are taking a workaround approach to fix the test.
However, there is also a way to change the config file.

Comment (feeblefakie) [reply test/src/integration/integration_test.go:173]: Thank you!
Just a question. I noticed we need the `monitoring` namespace from a certain point but is the name specified by the Prometheus operator we use?

I think we can take the following three ways basically?
1. Take the current approach (done in this PR)
1. Disable monitoring
1. Use the default namespace for monitoring for this test (not sure if it is possible)

So, first of all, my understanding is correct?
We should check if monitoring is deployed properly so we should take 1 or 3?
3 is possible?

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

I left one question. Let's decide the next action based on what we can/should do.

File: test/src/integration/integration_test.go (hunk 1 above)
Comment (miseyu) [reply test/src/integration/integration_test.go:173]: @feeblefakie 

> So, first of all, my understanding is correct?

Yes, I am aware of that. :bow:

> We should check if monitoring is deployed properly so we should take 1 or 3?
> 3 is possible?

I think it is possible to change the namespace of prometheus by modifying this.
https://github.com/scalar-labs/helm-charts/blob/main/charts/scalardl/values.yaml#L77-L96
https://github.com/scalar-labs/scalar-kubernetes/blob/master/playbooks/roles/prometheus/defaults/main.yml#L2

**Pull request merged**